modules/ROOT/pages/push/apple-setup.adoc

```diff
@@ -3,71 +3,61 @@ include::{partialsdir}/attributes.adoc[]
 [[app-id-ssl-certificate-apns]]
 = Apple related setup
 
-Before you are able to use Push Notifications on your iOS Application, a few steps are required. This section of the tutorial gives a quick overview of what is needed!
+Before you are able to use {unifiedpush-service} on your iOS Application, a few steps are required. This section of the tutorial gives a quick overview of what is needed!
 
 == Certificate Signing Request
 
 First you need to submit a request for a new digital certificate, which is based on a public/private key. The Certificate itself is acting as the public key, and when you request it, a private key is added to your *KeyChain* tool. The Cerficate will be used later on, to request an SSL certificate for the _Apple Push Network Service_, which will allow the _AeroGear Push Server_ to send notification messages to it. Now the actual CSR (Certificate Signing Request) is done using *KeyChain Access* tool found in *Applications > Utilities* folder on your Mac.  Once opened, in the *KeyChain Access* menu, choose *Certificate Assistant > Request a Certificate from a Certificate Authority*:
 
-image:./img/SubmitCSR.png[Submit Signing Request]
+image::apns-submit-csr.png[Submit Signing Request]
 
 Make sure that you have choosen to store the CSR _on file_, so we can upload it later in the provisioning portal when requesting the actual SSL cert. Going back to *KeyChain Access* you now see a new *private* key:
 
-image:./img/KeyChain_prtKey.png[private]
+image::apns-keychain-prtkey.png[private]
 
 == Apple App ID and SSL certificate for APNs
 
 Now that the CSR is generated, go to the link:https://developer.apple.com/account/overview.action[Provisioning Portal] and log in with your Apple developer account. Now, click on the *Identifiers* link in order to create a new App ID (use the *PLUS* Icon on the right). In the formular give the *App ID* a _descriptive_ name, like *My first AeroGear Push App*. Double check that the *Push Notifications* checkbox is _selected_:
 
+image::apns-appid-1.png[Give the App ID a name]
 
-image:./img/AppID_1.png[Give the App ID a name]
+In the same page you are asked for an *Explicit App ID*, since the generic *Wildcard App ID* does not work with {unifiedpush-service}. In the _Bundle ID_ field enter *YOUR* bundle ID. This is similar to Java packages. In this example we choose _org.aerogear.PushTest_, however you *must* use your own ID. *NOTE:* The _Bundle ID_ has to match the one from the actual iOS application that you are building later in this guide.
```

Comment:
Instead of `*NOTE:*` use a new paragraph prefixed by `NOTE:`:

```
NOTE: The _Bundle ID_ has to match the one from the actual iOS application that you are building later in this guide.
```

This will render much nicer.